ArduCopter/GCS_Mavlink.cpp

```diff
@@ -995,30 +995,29 @@ MAV_RESULT GCS_MAVLINK_Copter::handle_command_long_packet(const mavlink_command_
 
 MAV_RESULT GCS_MAVLINK_Copter::handle_command_pause_continue(const mavlink_command_int_t &packet)
 {
-#if MODE_AUTO_ENABLED
-    if (copter.flightmode->mode_number() != Mode::Number::AUTO) {
-        // only supported in AUTO mode
+#if defined(MODE_GUIDED_ENABLED) || defined(MODE_AUTO_ENABLED)
```

Comment:
let's remove these #define lines.  Guided and Auto are almost always included and there are no specific calls to auto or guided anymore after this change.